EU4toV2/Data_Files/cultureMap.txt

```diff
@@ -1,1130 +1,1130 @@
-# Copyright (c) 2019 The Paradox Game Converters Project
```

Comment:
Something bad happened to this file. It needs to be fixed.

Probably the line endings got changed. How did you edit it? Notepad? Try opening it in Notepad++ and using Edit->EOL Conversion->Unix

Reply:
I used Notepad++, but don't worry, I can fix it in a sec

Reply:
Somehow it didn't work. The things I edited was to simply change all instances of `eu4 = manchu` to `eu4 = manchu_new eu4 = manchu eu4 = evenk`